src/coreclr/vm/peassembly.cpp

```diff
@@ -732,11 +732,11 @@ PEAssembly::PEAssembly(
         m_pHostAssembly = pBindResultInfo;
     }
 
-#if _DEBUG
+#ifdef LOGGING
     GetPathOrCodeBase(m_debugName);
     m_debugName.Normalize();
```

Comment:
```suggestion
```
Normalize makes the string encoded as Unicode. It is unnecessary since the next line will re-encode it as UTF8